locales/es.yaml

```diff
@@ -4,26 +4,26 @@ hello: ¡Hola Mundo!
 # Page titles
 page-title:
   home: Página principal
-  song: Datos de la canción
+  song: Datos de canción
   songs: Lista de canciones
-  gallery: Listas de pistas
+  gallery: Lista de pistas
   about: Acerca de
   bug-report: Reportar problemas
   source-code: Código fuente
 
 # General UIs
 ui:
-  ok: DE ACUERDO
+  ok: De acuerdo
   cancel: Cancelar
   close: Cerrar
-  search: Buscar en
-  preview: Vista Previa
+  search: Buscar
+  preview: Vista previa
   download: Descargar
   goBack: Atrás
-  all: TODAS
+  all: Todas
```

Comment:
This entry is used as the placeholder in the text field.
The English translation is uppercased, but you can keep it capitalized if it looks more natural in Spanish.